pywr-schema/src/model.rs

```diff
@@ -977,6 +1005,26 @@ mod tests {
             panic!("Expected an error due to missing file: {str}");
         }
     }
+
+    #[test]
+    fn test_v1_conversion() {
+        let v1_str = include_str!("./test_models/v1/timeseries.json");
+        let v1: pywr_v1_schema::PywrModel = serde_json::from_str(v1_str).unwrap();
+
+        let (v2, errors) = PywrModel::from_v1(v1);
+
+        assert_eq!(errors.len(), 0);
+
+        std::fs::write("tmp.json", serde_json::to_string_pretty(&v2).unwrap()).unwrap();
+
+        let v2_converted: serde_json::Value =
+            serde_json::from_str(&serde_json::to_string_pretty(&v2).unwrap()).unwrap();
+
+        let v2_expected: serde_json::Value =
+            serde_json::from_str(include_str!("./test_models/timeseries-converted.json")).unwrap();
```

Comment:
I think this should sit next to the v1 JSON file: `test_models/v1/timeseries-converted.json`

Reply:
done